### PR TITLE
hotFix-invest

### PR DESCRIPTION
**Pull request by ghost**
**Invest Page - Updates made**

- Resized images (smaller size)
- Aligned text and images
- Removed dark gradients
- Mobile friendly

invest1
invest2
invest3

### CODE REVIEW

File: components/invest/NftInvest.tsx [hunk 1]
```diff
@@ -38,12 +32,18 @@ function NftInvest() {
               display='flex'
               width={'90%'}
               justifyContent='center'>
-              <Button onClick={() => window.open('https://opensea.io/collection/chippi', '_blank')}>Invest in Chippi</Button>
+              <Button
```
Comment (activate-glacier-instinct) [inline components/invest/NftInvest.tsx:35]: MINOR: this looks a bit weird, probably legacy code. 

Might be better to use next/link instead

File: components/invest/GmiIndex.tsx [hunk 2]
```diff
@@ -46,6 +51,7 @@ function GmiIndex() {
             alignItems={'center'}
             flexDirection={{ base: 'column', md: 'row' }}>
             <Button
+              fontWeight={800}
               margin='10px 0px;'
               onClick={() =>
```
Comment (activate-glacier-instinct) [inline components/invest/GmiIndex.tsx:56]: MINOR: same `next/link` comment (see below)

File: components/invest/DaoPunkSection.tsx [hunk 3]
```diff
@@ -50,24 +53,22 @@ function DaoPunkSection() {
                 pushing the boundaries of where technology meets on-chain voting
                 and automatic treasury triggers.
               </Text>
+              <Container mt={8}>
+                <Button
+                  fontWeight={800}
+                  onClick={() =>
```
Comment (activate-glacier-instinct) [inline components/invest/DaoPunkSection.tsx:59]: MINOT : another instance of `next/link`

**Review by activate-glacier-instinct — APPROVED** [review]
1 minor comment - non-breaking, not required for release.

All good otherwise.

**Review by ghost — COMMENTED** [review]
Updated branch to dev

**Comment by CompositeFellow** [comment]
confirmed updated to dev.  Preview looks good

**Pull request merged**